### DISCUSSION TITLE
Any Gridview, Gridtable for Desktop Slint components ?

### DISCUSSION THREAD

**Discussion by TERIIHUB** [slint-ui/slint#5147] [General] [answered]
Hi,
noobi, looking for to know about any Gridview components in slint..and where to find resource about this ...
*I only find example of Gui for tiny windows project (beside the large audio studio) 
 Thanks for any tips...
![images](https://github.com/slint-ui/slint/assets/3276014/ae1e22e5-ee32-4a95-930d-ec88ab5378ae)

**Comment by ogoffart** [comment, marked as answer]
There is a StandardTableView. There is a spreadsheet example in 7guis: https://github.com/slint-ui/slint/tree/master/examples/7guis#cells
Also i think coop_widget has an implementation of a TableView which is a bit more rich: https://codeberg.org/flovansl/co_sl/src/branch/main/widgets/coop/  
Another one: https://surrealism-all.github.io/SurrealismUI.github.io/components/data/table.html
